zebra-chain/src/work/difficulty.rs

```diff
@@ -257,6 +267,11 @@ impl CompactDifficulty {
         let expanded = self.to_expanded()?;
         Work::try_from(expanded).ok()
     }
+
+    /// Returns the raw inner value.
+    pub fn to_value(self) -> u32 {
```

Comment:
Should this be feature flagged?

Optional: I think a copy op would be ok here

```suggestion
    #[cfg(feature = "getblocktemplate-rpcs")]
    pub fn to_value(&self) -> u32 {
```

Reply:
yea, added that in https://github.com/ZcashFoundation/zebra/pull/5659/commits/d30e5e4143063d685f140697a08c0f95cf429147